### ISSUE TITLE
Single-Winner Threshold Selection

### ISSUE THREAD

**Issue by rrojas350** [BrightSpots/rcv#598]
For single-winner RCV, Minneapolis would like the ability to set the threshold based on the number of valid votes in the first round of counting, rather than having the threshold change each round. Similar to how the threshold is calculated in multi-winner already.

**Comment by tarheel** [comment]
If I'm understanding the proposed rule correctly, this would have no impact on the eventual winner; it would just mean that in some instances you might run additional rounds of tabulation even after you've conclusively determined a winner (kind of like our existing "continue until two candidates remain" option). Is that right?

**Comment by chughes297** [comment]
Yeah, no change to who wins. Essentially just a change to how the results look at the end.

**Comment by tarheel** [comment]
Right. But in this case, we would still declare the candidate with more votes to be the winner once there are only two candidates remaining.
